### PR TITLE
TGP-2948: Directs to single coods page now, start on tests

### PR DESCRIPTION
**Pull request by besscerule**
# Checklist for developers before reviewing
 - [x] I have merged in main
 - [x] I have assigned the PR to me
 - [x] I ran the code with a clean compile test and ensured there are no warnings or deprecations
 - [x] I have run `sbt prePR` to run all tests, format and check the code coverage
 - [x] I have written unit tests to cover any code that I wrote, where necessary
 - [x] I have linked tickets to any TODOs I have written
 - [x] I have run the microservices with the latest SM2 config
 - [x] I have checked all the Acceptance Criteria pass
 - [x] I have run the pipeline journey tests
 - [x] I have spoken to the QAs about any changes that may need to be made to the journey tests OR no changes to the journey tests are necessary
 - [x] I have moved the ticket to Review in Jira
 - [x] I have left a message in tgp-prs on slack to alert other Devs that this ticket is ready for review
 - [x] I have checked my PR is not in draft mode

# Checklist for reviewer 1
- [ ] I have added an emoji to the relevant message in tgp-prs to show I am reviewing
- [ ] I have checked out the branch
- [ ] I have run the microservices with the latest SM2 config
- [ ] I have checked all the Acceptance Criteria pass
- [ ] I reviewed the code and left comments where necessary
- [ ] I have left a message on the relevant tgp-prs message to show that I have finished reviewing

# Checklist for reviewer 2
- [ ] I have added an emoji to the relevant message in tgp-prs to show I am reviewing
- [ ] I have checked out the branch
- [ ] I have run the microservices with the latest SM2 config
- [ ] I have checked all the Acceptance Criteria pass
- [ ] I reviewed the code and left comments where necessary
- [ ] I have left a message on the relevant tgp-prs message to show that I have finished reviewing

# Checklist for developers before merging
 - [ ] The PR has been approved by 2 other developers
 - [ ] I have merged in main
 - [ ] I have run `sbt preMerge` to run all tests, check formatting and code coverage
 - [ ] I have run the microservices with the latest SM2 config
 - [ ] I have checked all the Acceptance Criteria pass
 - [ ] I have checked that the QAs have merged in any changes to the journey tests
 - [ ] I have run the pipeline journey tests

# Reminders for developers after merging
 1. Wait for pipeline to pass
 2. Move ticket to QA column in Jira
 3. Leave message in tgp-qa on slack to alert QAs that this ticket is ready for testing

### CODE REVIEW

File: app/controllers/goodsRecord/SingleRecordController.scala [hunk 1]
```diff
@@ -140,12 +139,24 @@ class SingleRecordController @Inject() (
...
-            para
+            para,
+            record.declarable,
+            getReviewReasonMessageKey(record.toReview, record.reviewReason)
           )
         ).removingFromSession(initialValueOfHasSuppUnit, initialValueOfSuppUnit)
       }
     }
 
+  private def getReviewReasonMessageKey(toReview: Boolean, reviewReason: Option[String]): Option[String] =
```
Comment (Reece-Carruthers) [inline app/controllers/goodsRecord/SingleRecordController.scala]: I thought we were getting rid of this in place of reading the JSON string into relevant case models which we can use within the view? e.g https://github.com/hmrc/trader-goods-profiles-frontend/pull/612/files#diff-0a81a36ebf9cb747ed7bdcad81c16fa5efbdd3c38562a5179e273749bdb31d79R29

**Review by Ewan-James-Donovan — DISMISSED** [review]

**Review by Reece-Carruthers — DISMISSED** [review]
LGTM :+1:

File: app/views/goodsRecord/SingleRecordView.scala.html [hunk 2]
```diff
@@ -49,12 +51,39 @@ <h3 class="govuk-notification-banner__heading">
...
+            <strong class="govuk-tag govuk-tag--red">
+                @messages(declarableStatus.messageKey)
+            </strong>
+        </td>
+    </p>
     @if(recordLocked){
         <p class="govuk-inset-text">@messages("singleRecord.recordLocked")</p>
-    } else {
-        <p class="govuk-body">@messages("singleRecord.p1")</p>
-        <p class="govuk-body">@messages("singleRecord.p2")</p>
```
Comment (yohan-oc) [inline app/views/goodsRecord/SingleRecordView.scala.html:57]: Should we remove these from messages file?

Comment (omotrium) [reply app/views/goodsRecord/SingleRecordView.scala.html:57]: yes, I just removed it from the files

**Review by yohan-oc — APPROVED** [review]
LGTM

**Pull request merged**